src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -2112,6 +2146,7 @@ def remove_sync_flow(self, group_id: str, flow_id: str, flow_type: str,
         if bucket_name:
             rgw_sync_policy_cmd += ['--bucket', bucket_name]
 
+        logger.info("Removing sync flow! %s", rgw_sync_policy_cmd)
```

Comment:
I believe f strings are preferred as they are usually easier to work with and easier to read

```suggestion
        logger.info(f'Removing sync flow {rgw_sync_policy_cmd}')
```

Reply:
if I do this, then I get some pylint error, so kept the same as it was before

Reply:
Whats the pylint error ?